tests/ser_de.rs

```diff
@@ -97,6 +101,25 @@ fn get_test_tuples() -> Vec<(String, SignedRawInvoice, Option<SemanticError>)> {
 					)
 				}).unwrap(),
 			None
+		),
+		(
+			"lnbc20m1pvjluezpp5qqqsyqcyq5rqwzqfqqqsyqcyq5rqwzqfqqqsyqcyq5rqwzqfqypqdq5vdhkven9v5sxyetpdeessp59g4z52329g4z52329g4z52329g4z52329g4z52329g4z52329g4qv56atym9qj3ucaz3zvkgv37jfd8yrwzv5pxzp7qlllmwt966kz4qvunzmdznvz5czkwu5lwvpcjz6lhn4n20ff0ns5khwvafh0ufs2qph75xhc".to_owned(),
+			InvoiceBuilder::new(Currency::Bitcoin)
+				.payment_hash(sha256::Hash::from_hex(
+					"0001020304050607080900010203040506070809000102030405060708090102"
+				).unwrap())
+				.description("coffee beans".to_string())
+				.amount_pico_btc(20000000000)
+				.timestamp(UNIX_EPOCH + Duration::from_secs(1496314658))
+				.payment_secret(PaymentSecret([42; 32]))
+				.build_signed(|msg_hash| {
+					let privkey = SecretKey::from_slice(&hex::decode("e126f68f7eafcc8b74f54d269fe206be715000f94dac067d1c04a8ca3b2db734").unwrap()).unwrap();
```

Comment:
`SecretKey` implements `FromStr`, which parses a hex string as a secret key. So `hex` isn't needed as an external dependency.

As a side note: While I think just parsing a signature from hex would be nicer, signing should work as the nonces are deterministic afaik.